src/v0/destinations/marketo_bulk_upload/util.js

```diff
@@ -82,13 +91,9 @@ const getAccessToken = async config => {
         resp
       );
     }
-    throw new NetworkInstrumentationError(
-      "Could not retrieve authorization token"
-    );
+    throw new UnauthorizedError("Could not retrieve authorization token");
   }
-  throw new NetworkInstrumentationError(
-    "Could not retrieve authorization token"
-  );
+  throw new UnauthorizedError("Could not retrieve authorization token");
```

Comment:
These are not unauthorized errors. You can treat them as general network errors.